assets/dev/scss/frontend/widgets/playing-cards.scss

```diff
@@ -0,0 +1,213 @@
+.elementor-widget-playing-cards {
+	--e-card-width: 220px;
+	--e-cards-flex-direction: row;
+	--e-cards-flex-justify-content: start;
+	--e-cards-flex-align-items: start;
+	--e-cards-transition-time: 300ms;
+	--e-cards-row-gap: 10px;
+	--e-cards-col-gap: 10px;
+	--e-card-border-radius: 10px;
+	--e-card-border: .7px solid #ccc;
+	--e-card-font-family: initial;
+	--e-card-font-size: 25px;
+	--e-card-font-weight: 300;
+	--e-card-diamond-color: red;
+	--e-card-heart-color: red;
+	--e-card-spade-color: black;
+	--e-card-club-color: black;
+	--e-card-suit-font-size: 35px;
+	--e-card-royal-suit-offset-h: 10%;
+	--e-card-royal-suit-offset-v: 25%;
+	--e-card-background-color: white;
+	--e-card-padding: 2px 4px;
+
+	.elementor-widget-container {
+		display: flex;
+		flex-wrap: wrap;
+		flex-direction: var(--e-cards-flex-direction);
+		justify-content: var(--e-cards-flex-justify-content);
+		align-items: var(--e-cards-flex-align-items);
+		row-gap: var(--e-cards-row-gap);
+		column-gap: var(--e-cards-col-gap);
+		transition: var(--e-cards-transition-time);
+
+		.e-playing-cards-item {
+			transition: var(--e-cards-transition-time);
+			width: var(--e-card-width);
+			height: calc(var(--e-card-width) / 0.7);
+			border-radius: var(--e-card-border-radius);
+			box-shadow: rgba(0, 0, 0, 0.1) 0px 4px 12px;
+			background-color: var(--e-card-background-color);
+			transform-style: preserve-3d;
+			cursor: pointer;
+			border: var(--e-card-border);
+
+			* {
+				pointer-events: none;
+				user-select: none;
+			}
+
+			.e-card-backside {
+				position: relative;
+				position: absolute;
+				width: 100%;
+				height: 100%;
+				border-radius: inherit;
+				background-color: inherit;
+				transform: rotateY(.5turn);
+				padding: 10px;
+				overflow: hidden;
+
+				&-image {
+					background-image: url(../images/playing-cards/backside.jpg);
+					background-size: cover;
+					width: 100%;
+					height: 100%;
+				}
+			}
+
+			&-flipped {
+				transform: rotateY(.5turn);
+
+				.e-card-backside {
+					transform: rotateY(1turn);
+				}
+			}
+
+
+			&.heart {
```

Comment:
In a project, I would expect something like: `.e-card-heart` or `e-suit-heart.`
We try to avoid standalone class names, so that we can structure the code better.

I am happy with this in this context.